docs/security/host-certs.md

```diff
@@ -205,6 +205,56 @@ For instructions on how to request a host or service certificate using the Web
 interface please see the [user guide maintained by
 the OIM development team](https://confluence.grid.iu.edu/pages/viewpage.action?pageId=3244064).
 
+Requesting Host Certificate for HTCondor-CE Using Let's Encrypt
+---------------------------------------------------------------
+
+As an alternative to the above options, the Let's Encrypt software can be used to obtain host certificates.
+
+We use the following guide as a reference:
+
+   - https://github.com/cilogon/letsencrypt-certificates
+
+In particular, under Getting your host certificate, we follow the two "Setting up" sections.
+
+(See also: https://letsencrypt.org/getting-started/)
+
+The `letsencrypt` software (AKA `certbot`) can be obtained from the EPEL 7 yum repo:
+
+        :::console
+        root@host # yum install certbot
+
+If you already have an HTCondor-CE set up and running, stop the CE View service (as it listens on
+port 80, which the letsencrypt setup needs to bind on temporarily).
+
+From the git checkout, you can run the following command to obtain the host certificate
+with Let's Encrypt:
+
+        :::console
+        root@host # certbot certonly --standalone --email %RED%<ADMIN_EMAIL>%ENDCOLOR% -d %RED%<HOST>%ENDCOLOR%
+
+
+Set up hostcert/hostkey links:
+
+        :::console
+        root@host # ln -s /etc/letsencrypt/live/*/cert.pem /etc/grid-security/hostcert.pem
+        root@host # ln -s /etc/letsencrypt/live/*/privkey.pem /etc/grid-security/hostkey.pem
+        root@host # chmod 0600 /etc/letsencrypt/archive/*/privkey*.pem
+
+Set up /etc/grid-security/certificates:
+
+        :::console
+        root@host # git clone https://github.com/cilogon/letsencrypt-certificates.git
+        root@host # cd letsencrypt-certificates/
+        root@host # make check
+        root@host # make install
+
+
+Before the hostcert expires, you can renew it with:
+
+        :::console
+        root@host # certbot renew
+
```

Comment:
Is there a way to automate renewal? These certs are very short-lived compared to the host certs that our users are familiar with

Reply:
How about
```
* * 1 * * root certbot renew
```

Reply:
I see this file which suggests auto-renewal functionality:

https://github.com/certbot/certbot/blob/master/certbot/renewal.py#L1

but i could not find anything obvious telling how that works.  (As far as i can tell, it only happens when `certbot renew` is run, which of course admins can automate with their favorite method, if they like.)